helpdesk_fieldservice/views/helpdesk_ticket_views.xml

```diff
@@ -45,4 +45,20 @@
             </field>
         </field>
     </record>
+
+    <menuitem id="menu_helpdesk_op"
+              name="Operations"
+              parent="helpdesk.menu_helpdesk_root"
+              sequence="10"/>
+
+    <menuitem id="menu_helpdesk_product"
+              name="Service Orders"
+              action="fieldservice.action_fsm_dash_order"
+              parent="menu_helpdesk_op"
+              sequence="20"/>
+
+    <menuitem id="helpdesk.helpdesk_ticket_menu_main" 
+        name="All Tickets"
```

Comment:
```suggestion
```

Reply:
If we don't have that the menu shows as 'helpdesk.helpdesk_ticket_menu_main' where the name should be

Reply:
![Helpdesk](https://user-images.githubusercontent.com/39602586/58726139-c893b580-8395-11e9-88e5-ca6ff04358ca.PNG)